src/mmw/js/src/compare/templates/compareWindow.html

```diff
@@ -1,8 +1,5 @@
 <div id="compare-scenarios-region"></div>
 <div id="compare-footer">
-    <div class="back"> <!-- Back -->
-      <a data-url="/project" class="btn btn-md btn-icon"><i class="fa fa-arrow-left"></i></a>
-    </div> <!-- End Back -->
     <h3>Navigate Scenarios</h3>
```

Comment:
Should we remove this or is it referring to the arrows on the far right of the page? Either way I find it confusing.

![screen shot 2017-03-08 at 12 05 26 pm](https://cloud.githubusercontent.com/assets/7633670/23714589/d1523c70-03f7-11e7-81cb-b3fbaef3e9d2.png)